### PR TITLE
add query history DB Manager

### LINKED ISSUE #2607: [FEATURE][needs-docs] DB Manager: adds SQL query history
Original commit: https://github.com/qgis/QGIS/commit/1242402a561cf963be5a50a48eb10ac350dfd314 by slarosa

Unfortunately this naughty coder did not write a description... :-(

### PR DESCRIPTION
**Pull request by ghtmtt**
Goal: add the description of the `Query History` button of DB Manager. I didn't add any screenshot (not worth it IMHO)

Ticket(s): fixes #2607

- [x] Backport to LTR documentation is required

### CODE REVIEW

File: docs/user_manual/plugins/core_plugins/plugins_db_manager.rst [hunk 1]
```diff
@@ -90,6 +90,10 @@ button.
    See :ref:`vector_virtual_layers` for instructions on the SQL syntax
    to use.
 
+The :guilabel:`Query History` button stores the last 20 queries of each database
+and provider.
```
Comment (DelazJ) [inline docs/user_manual/plugins/core_plugins/plugins_db_manager.rst:94]: That means that if I have 3 PG database connections I could store up to 60 queries?
maybe nitpick :thinking:  but does that button "store" or "display saved queries"

**Review by DelazJ — COMMENTED** [review]
Thanks @ghtmtt 
Could we move the changes above the screenshot? We mention (slightly) query saving and loading there so they'd complete. That paragraph may need split but i'm not asking ;)

File: docs/user_manual/plugins/core_plugins/plugins_db_manager.rst (hunk 1 above)
Comment (ghtmtt) [reply docs/user_manual/plugins/core_plugins/plugins_db_manager.rst:94]: Store and/or Saved queries allows you to save the queries within the project itself and give a particular name to the query in order to get it fast

**Comment by ghtmtt** [comment]
Thanks for the review @DelazJ. I moved the section above the screenshot and rephrased the sentences. If we don't have any other objections I'll merge this :)

**Review by havatv — APPROVED** [review]
Looks fine to me.

**Pull request merged**